src/main/java/com/github/luben/zstd/Zstd.java

```diff
@@ -757,6 +757,7 @@ public static long trainFromBufferDirect(ByteBuffer samples, int[] sampleSizes,
     public static native int frameHeaderSizeMin();
     public static native int frameHeaderSizeMax();
     public static native int blockSizeMax();
+    public static native int defaultCompressionLevel();
     /* Min/max compression levels */
     public static native int minCompressionLevel();
     public static native int maxCompressionLevel();
```

Comment:
I didn't group `defaultCompressionLevel()` with `minCompressionLevel()` and  `maxCompressionLevel()` because its value is derived from a constant, which matches the preceding section ("Constants from the zstd_static header").